bitbots_body_behavior/bitbots_body_behavior/actions/go_to_defense_position.py

```diff
@@ -59,7 +59,9 @@ def perform(self, reevaluate=False):
             pose_msg.pose.position.x = defense_pos[0]
             pose_msg.pose.position.y = defense_pos[1]
             yaw = math.atan(-vector_ball_to_goal[1] / -vector_ball_to_goal[0])
-            pose_msg.pose.orientation = Quaternion(*quaternion_from_euler(0, 0, yaw))
+            x, y, z, w = quaternion_from_euler(0, 0, yaw)
+            pose_msg.pose.orientation = Quaternion(x=x, y=y, z=z, w=w)
```

Comment:
What was the reason for this Change? For me, both versions look the same...😅

Reply:
ROS2 does not support positional arguments for the message fields anymore. They only support keyword arguments.